__test__/components/indexPage.test.js

```diff
@@ -143,7 +143,7 @@ describe('Index subscribe form', () => {
     expect(indexForm.state().email).toEqual('yinks@gmail.com');
   });
 });
-describe('Index subscribe form', () => {
+describe('Index carousel', () => {
```

Comment:
You actually made a change you know 😏

Reply:
It has to be from the rebase

Reply:
But still, this is a string in the describe block, it shouldn't affect the test.

Reply:
Yeah I know hence the smirk.